docs/source/conf.py

```diff
@@ -17,7 +17,7 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'IBM Spectrum Scale CSI Operator'
+project = 'IBM Spectrum Scale CSI'
 copyright = '2019, IBM'
 author = 'John Dunham'
```

Comment:
Eventually we might want to change this author  haha.

Reply:
Can we just leave it blank?